dockerfile/ubuntu-20.04-x86.Dockerfile

```diff
@@ -0,0 +1,40 @@
+# Second stage: the actual image
+FROM ubuntu:20.04
+
+ARG DEBIAN_FRONTEND=noninteractive
+ENV GTEST_VERSION=1.13.0
+ENV DOXYGEN_VERSION=1.9.6
+
+# Install build and runtime deps
+COPY /scripts/docker/requirements.txt /opt/tt_metal_infra/scripts/docker/requirements.txt
+RUN apt-get -y update \
+    && xargs -a /opt/tt_metal_infra/scripts/docker/requirements.txt apt-get install -y --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/*
+
+# Install dev deps
+COPY /scripts/docker/requirements_dev.txt /opt/tt_metal_infra/scripts/docker/requirements_dev.txt
+RUN apt-get -y update \
+    && xargs -a /opt/tt_metal_infra/scripts/docker/requirements_dev.txt apt-get install -y --no-install-recommends \
+    && rm -rf /var/lib/apt/lists/*
+
+## Test Related Dependencies
+COPY /scripts/docker/install_test_deps.sh /opt/tt_metal_infra/scripts/docker/install_test_deps.sh
+RUN /bin/bash /opt/tt_metal_infra/scripts/docker/install_test_deps.sh ${GTEST_VERSION} ${DOXYGEN_VERSION}
+
+# Copy remaining convenience scripts
+COPY /scripts /opt/tt_metal_infra/scripts
+COPY build_metal.sh /scripts/build_metal.sh
+
+# ENV TT_METAL_INFRA_DIR=/opt/tt_metal_infra
```

Comment:
Can we delete this extra commented stuff?

Reply:
Will be for future work in including Virtual Env in Docker. This will be integrated but just disabled for future reference